www/config.yml

```diff
@@ -766,3 +766,14 @@ chapters:
       location:
             latitude: 38.8949549
             longitude: -77.0366456
+
+    - name: Kampala, Uganda
+      meetup_id: 244079349
+      website: uganda
```

Comment:
Hey @jnanjekye - this value should match the directory that you've added `www/locations/kampala/index.html`. So either this `website:` value should be `kampala`, or the path you've added should be `www/locations/uganda/index.html`.